sysdig-values.yaml

```diff
@@ -26,8 +26,8 @@ agent:
   #       proxy_user: smartproxydev
   #       proxy_password: dsnysv_137
   #       ssl: false
-  image:
-    tag: 12.3.1
+  # image:
+  #   tag: 12.3.1
```

Comment:
```suggestion
```
We can remove the yaml for image/tag completely. No need to leave the commented out lines. Ideally in 99% of use-cases the customer should allow these values to be driven by the chart default values.